primitives/src/v2/mod.rs

```diff
@@ -1403,6 +1403,18 @@ pub struct DisputeState<N = BlockNumber> {
 	pub concluded_at: Option<N>,
 }
 
+#[cfg(feature = "std")]
+impl MallocSizeOf for DisputeState {
```

Comment:
Is there a better way to implement this trait?

Reply:
I would suggest to add `#[cfg_attr(feature = "std", derive(MallocSizeOf))]` attribute for `DisputeState` structure definition.

Reply:
I tried this initially but `BitVec<T>` doesn't implement `MallocSizeOf`:
```
error[E0277]: the trait bound `BitVec<u8>: MallocSizeOf` is not satisfied
    --> primitives/src/v2/mod.rs:1395:36
     |
1395 | #[cfg_attr(feature = "std", derive(MallocSizeOf))]
     |                                    ^^^^^^^^^^^^ the trait `MallocSizeOf` is not implemented for `BitVec<u8>`
     |
     = note: this error originates in the derive macro `MallocSizeOf` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Reply:
So you are fine with what I've done here considering the circumstances?

Reply:
I am as long as it's computed correctly. I don't think it currently is though. Try running this snippet
```rust
  1 use bitvec::bitvec;                                                             
  2                                                                                 
  3 fn main() {                                                                     
  4     let b = bitvec![u8, bitvec::order::Lsb0; 1; 4000];                          
  5     println!("{}", std::mem::size_of_val(&b));                                  
  6 }      
  ```